homeassistant/components/binary_sensor/rflink.py

```diff
@@ -5,13 +5,15 @@
 https://home-assistant.io/components/binary_sensor.rflink/
 """
 import logging
+import voluptuous as vol
```

Comment:
Add a blank line between standard library and 3rd party imports.

Reply:
OK I'll fix, but why isn't this picked up by the lint tools?